Pod/Classes/GraphQL/ARGraphQLQueryCache.m

```diff
@@ -3,13 +3,14 @@
 #import <CommonCrypto/CommonDigest.h>
 
 #ifdef DEBUG
+// No caching in dev mode.
+const NSTimeInterval ARGraphQLQueryCacheDefaultTTL = 0;
 #define DLog(...) NSLog(__VA_ARGS__)
 #else
+const NSTimeInterval ARGraphQLQueryCacheDefaultTTL = 3600;
 #define DLog(...) /* */
 #endif
 
-const NSTimeInterval ARGraphQLQueryCacheDefaultTTL = 3600;
```

Comment:
@orta do you think I should leave this change in?

Reply:
nah, let's ditch it. Could have been a red herring.